.github/workflows/manage-github-issue-for-outdated-pods.yml

```diff
@@ -0,0 +1,34 @@
+name: Check outdate pod and create issue
+
+on: 
+  schedule:
+    - cron: '0 0 * * 1'
+  workflow_dispatch:
+
+jobs:
+  check-outdated-pods:
+    runs-on: macos-latest
+    name: Check outdate pods and create issue if it doesn't exist
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Check outdated pods and create issue
+        id: check-outdated-pods-and-create-issue
+        uses: rudderlabs/github-action-updated-pods-notifier@main
+        with:
+          outdated-pod-names: "Amplitude"
+          directory: "Example"
+          title: "fix: update Amplitude SDK to the latest version"
+          assignee: "desusai7"
+          labels: "outdatedPod"
+          color: "FBCA04"
+        env:
+          GH_TOKEN: ${{ github.token }}
+
+      - name: Get the github issue url
```

Comment:
Do we need these steps? It is just logging the URL.

Reply:
- Token is required in the [github-action-updated-pods-notifier](https://github.com/rudderlabs/github-action-updated-pods-notifier) to handle `issue`.
- For example refer [this](https://github.com/rudderlabs/github-action-updated-pods-notifier/blob/8d048f679cb0dde246ec29c9bc85536f820bccdb/scripts/multiple_outdatedpod.sh#L56) code. It runs `gh` command and it requires GitHub token to be present else that command will fail.
- Also refer [this](https://cli.github.com/manual/#:~:text=Run%20gh%20auth%20login%20to%20authenticate%20with%20your%20GitHub%20account.%20Alternatively%2C%20gh%20will%20respect%20the%20GITHUB_TOKEN%20environment%20variable.) doc.

Reply:
For line no. 28:

- It'll help in easily accessing the Issue URL on which this action has worked.